olp-cpp-sdk-core/include/olp/core/http/Network.h

```diff
@@ -83,5 +83,10 @@ class CORE_API Network {
   virtual void Cancel(RequestId id) = 0;
 };
 
+/**
+ * @brief Get default Network implementation.
+ */
+CORE_API std::unique_ptr<Network> GetDefaultNetwork();
```

Comment:
```suggestion
CORE_API std::unique_ptr<Network> CreateDefaultNetwork();
```
I think the "CreateDefaultNetwork" better describe what this method does, i.e. creating a new instance of the network class instead retrieving some pre-created object.